src/components/card/card.interface.ts

```diff
@@ -1,6 +1,7 @@
-import { plantDataType } from "@/shared/types";
+import { plantDataType } from "@/shared/type/data-types";
 
 export interface ICardProps {
   data: plantDataType;
   session?: any;
+  openLoginModal: () => void;
```

Comment:
🦆 to make this method agnostic to the modal, we can rename this to `onLikeClick`, in this way, the UI props are not defined by the configuration that happens on the frontend. In this example this is understood like, the heart can open a login modal, or it can register the click an entry in the backend.

Reply:
🥔 I updated all the name for this props as well 🐔